appserver/admin/cli-optional/src/main/java/com/sun/enterprise/admin/cli/optional/H2Control.java

```diff
@@ -90,14 +90,14 @@ public H2Control(final String dbCommand, final String dbHost, final String dbPor
     private void invokeServer() {
         try {
             Class serverClass = Class.forName("org.h2.tools.Server");
-            String dbPassword = getDbPassword() == null ? "" : getDbPassword();
+            String dbPassword = getDbPassword() == null ? "APP" : getDbPassword();
             String url = "tcp://localhost:" + getDbPort();
             if (null != getDbCommand()) 
                 switch (getDbCommand()) {
                     case "start": {
                         Method createTcpServer = serverClass.getDeclaredMethod("createTcpServer", 
                                 new Class[]{String[].class});
-                        Object[] paramObj = new Object[]{new String[]{"-tcpPort", getDbPort(), "-tcpPassword", dbPassword, "-tcpAllowOthers"}};
+                        Object[] paramObj = new Object[]{new String[]{"-tcpPort", getDbPort(), "-tcpPassword", dbPassword, "-tcpAllowOthers", "-ifNotExists"}};
```

Comment:
The combination of -ifNotExists with -tcpAllowOthers, -pgAllowOthers, or -webAllowOthers effectively creates a remote security hole in your system. More info here http://www.h2database.com/html/tutorial.html#creating_new_databases

Reply:
🤔

Reply:
Do we need `tcpAllowOthers` as only Payara server itself (on the same machine) needs to access H2. (And yes, users will no longer connect remotely to it with a database client for example)

Reply:
Removed tcpAllowOthers, not sure what effect this will have though.